src/components/ShopingCart/ShopingCart.jsx

```diff
@@ -92,7 +92,14 @@ const ShopingCartCourses = [
     rating: 4.5,
   },
 ];
+
+// calculate Total Price
+function calculateTotalPrice(courses) {
```

Comment:
We don't need to make the code more complex by adding a function to calculate the total price (reduce is the function itself), It will be enough to add the first line of this function inside the component after fetching all courses for example, or anywhere inside the component, and then render it in the JSX.
try it again my friend.